corehq/apps/domain/views/base.py

```diff
@@ -17,23 +17,31 @@
 from corehq.apps.users.models import SQLInvitation
 
 
+def covid19(request):
```

Comment:
Adding this view means that the `request.GET.get('next_view')` on line 35 below isn't going to be used. However, I'm keeping the URL param option because we've had other use cases for this functionality.